packages/typescript-checker/test/unit/typescript-helpers.spec.ts

```diff
@@ -101,13 +101,7 @@ describe('typescript-helpers', () => {
             false
           )
         ).compilerOptions
-      ).deep.include({
-        noEmit: true,
-        incremental: false,
-        composite: false,
-        declaration: false,
-        declarationMap: false,
-      });
+      ).not.hasOwnProperty('declarationDir');
```

Comment:
Great find! So we had a test for it, but it never worked correctly? 😢

Reply:
Yes, I just spotted this as well, because previous test only look for what is there, but not check what should missing.

Reply:
🤦‍♂️

Reply:
Serves me right for not adding an integration test when I implemented this fix.